plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -0,0 +1,1078 @@
+package admission
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+
+	kadmission "k8s.io/kubernetes/pkg/admission"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/auth/user"
+	"k8s.io/kubernetes/pkg/client/cache"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	clientsetfake "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/fake"
+	kscc "k8s.io/kubernetes/pkg/securitycontextconstraints"
```

Comment:
I take it that renaming this import and other uses of SCC is part of the WIP for this PR?